IBX-6709: Page Builder amendments / Drag and drop elements (#239)
ibexa/documentation-user: docs/content_management/create_edit_pages.md

# Create and edit Pages

> **Page** is a special Content Type.

Author: Not in the scope of this PR, will be covered in separated PR.

> You can create and modify Pages with a visual drag-and-drop editor - Page Builder.

Reviewer: ```suggestion
You can create and modify Pages with a visual drag-and-drop editor, the Page Builder.
```

> Page is divided into zones into which you can drop various dynamic blocks.
> By editing pages you can customize the layout and content of your website.

Reviewer: ```suggestion
Pages are divided into zones into which you can drop various dynamic blocks.
By editing pages, you can customize the layout and content of your website.
```

Whenever you edit a Page, a [new version](content_versions.md) is created in the repository.

!!! tip

>     The Page Content Type contains a **Landing Page** Field Type which manages

Author: Not in the scope of this PR, will be covered in separated PR.

## Edit Page

You can edit Pages using the Page Builder.
To start editing an existing Page, in the Back Office from the Content Tree choose the page that you want you edit and click **Edit** button.

![Selecting Page for editing](img/editing_page.png)

### Page Builder interface

Page Builder has plain and intuitive interface. It is easy to create a Page without having advanced technical skills.

![Page Builder interface](page_builder_interface.png)

Page Builder user interface consists of:

A. Drop zone

B. Elements / Structure view toolbar

C. Settings toolbar (including Fields, Visibility and Schedule settings)

> D. Mode toolbar (including Desktop, Tablet and Mobile mode)

Reviewer 2: Maybe Device toolbar?
